src/instructions/proctored_exam/prerequisites-instructions/index.jsx

```diff
@@ -9,17 +9,15 @@ import Footer from '../Footer';
 
 const PrerequisitesProctoredExamInstructions = ({ skipProctoredExam }) => {
   const state = useContext(ExamStateContext);
-  const { exam, proctoringSettings, allowProctoringOptOut } = state;
+  const { exam, allowProctoringOptOut } = state;
   const { prerequisite_status: prerequisitesData } = exam;
   const { pending_prerequisites: pending, failed_prerequisites: failed } = prerequisitesData;
-  const { platform_name: platformName } = proctoringSettings;
```

Comment:
can remove this completely because it was being passed in as a prop to `FailedPrerequisitesProctoredExamInstructions`